cmd/translate/service/service.go

```diff
@@ -71,9 +71,25 @@ var rootCmd = &cobra.Command{
 		// Gracefully stops GRPC server.
 		defer grpcServer.GracefulStop()
 
+		dbType := viper.GetString("service.db")
+		badgerDir := viper.GetString("db.badgerdb.path")
+		badgerTmpDir := os.TempDir() + "badgerdb"
+
+		if dbType == repo.BadgerDB && badgerDir == badgerTmpDir {
+			log.Printf("INFO: Using temporary storage directory '%s' for badgerdb\n", badgerTmpDir)
+
+			defer func() {
+				if rmErr := os.RemoveAll(badgerDir); rmErr != nil {
+					log.Printf("WARNING: Failed to remove temporary storage directory '%s' for badgerdb: %v\n", badgerTmpDir, rmErr)
+					return
+				}
+				log.Printf("INFO: Deleting temporary storage directory '%s' for badgerdb\n", badgerTmpDir)
+			}()
+		}
```

Comment:
We don't require to remove the temporary directory at the end. All tests MUST be isolated anyway.